lib/rufo/command.rb

```diff
@@ -161,7 +173,7 @@ def format(code, dir)
   end
 
   def self.parse_options(argv)
-    want_check = false
+    exit_code, want_check = true, false
```

Comment:
Nitpick: I prefer a separate line for each variable assignment

Reply:
I'll change (I have no preference either way)